public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -102,6 +102,10 @@ const bannerModules = {
     path: 'environmentMoment/EnvironmentMomentBanner.js',
     name: 'EnvironmentMomentBanner',
   },
+  [BannerTemplate.UsEoyMomentBanner]: {
+    path: 'environmentMoment/UsEoyMomentBanner.js',
```

Comment:
I think this should be `usEoyMoment/UsEoyMomentBanner.js` (it should be the directory name in SDc)

Reply:
Thanks for spotting this Tom!